src/process.cr

```diff
@@ -129,19 +129,26 @@ class Process
     pid
   end
 
-  # The standard `IO` configuration of a process:
-  #
-  # * `nil`: use a pipe
-  # * `false`: no `IO` (`/dev/null`)
-  # * `true`: inherit from parent
-  # * `IO`: use the given `IO`
-  alias Stdio = Nil | Bool | IO
+  # How to redirect the standard input, output and error IO of a process.
+  enum Redirect
+    PIPE
+    CLOSE
+    INHERIT
+  end
+
+  PIPE    = Redirect::PIPE
```

Comment:
I add a proposal for naming constants above.

I suppose that documentation can be copied, but we can make  `#See whatever` too.

Or, may be, we can to `private PIPE = Redirect::PIPE`?